.github/PULL_REQUEST_TEMPLATE/rust.md

```diff
@@ -0,0 +1,24 @@
+Changes
+-------
+
+* IoT3 SDK:
+    * client: new awesome method
+    * exchange: fixed something that was broken
+    * mobility: new stuff
+    * transport:  update some message version
+* examples:
+    * collector: add a new exporter
+    * copycat: manage the new awesome message type
+    * json_counter: change the frequency
+    * telemetry: add a span attribute
+
+Close #XXX
+
+Close #YYY
+
+Test
+---
+
+## How to test
```

Comment:
Same here, these are twice the same level

I'd rather put a single level 2 here as I would prefer inline expected results instead of splitting test instruction and expected results

```markdown
How to test
-----------

1. Do something
    **=> Expected result**
2. Test something else
    **=> Other expected result**
    **=> Expecting logs to be logged**
    > [INFO] I an expected log line  
```